tests/fv3_conf/ccpp_gfsv16_csawmg_run.IN

```diff
@@ -0,0 +1,40 @@
+
```

Comment:
I wonder if there is a possibility to reduce the number of run scripts for these similar tests in the future, i.e. combine some of `ccpp_csawmg_run.IN`, `ccpp_csawmgshoc_run.IN`, `ccpp_gfsv16_csawmg_run.IN` and possibly others. We would need to use some bash if blocks to query which configuration will be run, similar to what is done in `ccpp_gsd_run.IN`.